tests/unit/serve/executors/test_decorators.py

```diff
@@ -63,3 +64,67 @@ async def fn_3(*args, **kwargs):
     assert not iscoroutinefunction(getattr(fn_2, 'fn'))
     assert hasattr(fn_3, 'fn')
     assert iscoroutinefunction(getattr(fn_3, 'fn'))
+
+
+@pytest.mark.parametrize(
```

Comment:
make sure to be able to test the decorator without the Executor and move these tests to unit tests of Executor